step-functions/post-ivq-answers.asl.json

```diff
@@ -63,24 +63,25 @@
       },
       "ResultPath": "$.oAuthToken"
     },
-    "Do Nothing": {
+    "Nothing left to do": {
       "Type": "Pass",
-      "End": true
+      "End": true,
+      "Result": {}
```

Comment:
Is this significant?

Reply:
Yeah it's so the result of the state machine is `{}` and not the unneeded meta data from the other states